server/src/index.js

```diff
@@ -1,6 +1,41 @@
 import express from 'express';
+import mongoose from 'mongoose';
+import bodyParser from 'body-parser';
+import morgan from 'morgan';
+
+import router from './router';
+import { errorHandler, notFoundRoutes } from './libs/routes';
+import { PORT, MONGO_CONNECTION_STRING } from './configs/config';
```

Comment:
Should be used as environment variables. Consider using `dotenv` library and use `.env` instead.

Reply:
In the config file, they are being used as environment variables only.